setup_env.sh

```diff
@@ -68,3 +68,24 @@ else
         ${CONDA} env export --name ${ENV_NAME} --no-builds | \grep -v '^prefix:' > requirements/environment.yml || exit
     fi
 fi
+
+
+# Setting ECCODES_DEFINITION_PATH:
+${CONDA} activate ${ENV_NAME}
```

Comment:
Was this also tested with the `-m` option? I.e. where `CONDA=mamba`.

Reply:
hmm, I don't think so. so maybe better to remove this option?

Reply:
Mmmh, I am not sure if we ever use mamba for probtest. I guess that is coming from the blueprint, right?

Reply:
Judging from line 53 of this file, `${CONDA} activate/deactivate` is probably an issue when using mamba, so we should at least change those commands to just `conda activate/deactivate`. The rest is probably fine, judging from the code above.

However, if we don't care about mamba here, then I agree that we could actually remove it entirely.

Reply:
yes, it comes from the blueprint. Then lets keep it lean and remove the option.

Reply:
Ok :+1: